homework/Contributor.js

```diff
@@ -5,15 +5,28 @@
 // eslint-disable-next-line no-unused-vars
 class Contributor {
   constructor(contributor) {
-    this.contributor = contributor;
+    this.data = contributor;
   }
 
-  /**
-   * Render the contributor info to the DOM.
-   * @param {HTMLElement} container The container element in which to render the contributor.
-   */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.contributor, null, 2));
+    const contributorItemList = Util.createAndAppend('li', container, {
+      class: 'contributorItem',
+    });
+    contributorItemList.addEventListener('click', () => {
```

Comment:
This event listener will respond to mouse clicks but you can't visit the contributors url by moving to the contributor list item and then pressing Enter, as would be required for ARIA compliance.

Alternatively, you could generate an `a` tag as a direct child of the `li`, set its `href` to the contributor url and then create the remaining elements as children of the `a` tag. In this way the whole list element can be clicked. The Enter key will work too.